wiki/Modding/es.md

```diff
@@ -4,76 +4,43 @@ outdated_translation: true
 
 # Modding (Español)
 
-**Modding** es cuando alguien revisa el Beatmap de otra persona y hace sugerencias. Las sugerencias pueden ser para solucionar errores o simplemente para hacer el Beatmap mejor de lo que es. La persona haciendo el modding es llamado “[modder](/wiki/Modding/Modder)”, y la persona que hace el Beatmap es llamado “[mapper](/wiki/Beatmapping)”. Si tu mapeas, es altamente recomendado que des mod por tres grandes razones:
+*Para otros usos, revisar [Mod (desambiguación)](/wiki/Disambiguation/Mod).*
 
-1. Si tu mod post es útil para el mapper, entonces el te dará un [kudosu](/wiki/Modding/Kudosu). Los kudos pueden ser usados para aumentar el [Star Priority](/wiki/Modding/Star_priority) (SP) de tu map.
-2. “Mod4mods”, o cuando haces un trato con alguien para que haga un mod post en tu Beatmap a cambio de que tu hagas un mod post en el suyo.
-3. Un buen mod post reduce el tiempo que los equipos de aprobación utilizan señalando errores básicos. Esto a su vez reduce la cantidad de estrés en los equipos de aprobación, y adicionalmente hace que ellos estén más dispuestos a hacer un mod post a tu Beatmap. Confía en nosotros, un MAT feliz y de buen animo es el que no tuvo que gastar 10 minutos llorando porque el Beatmap obviamente necesitaba muchos más mod post.
+**Modding** es el acto de proveer retroalimentación al [beatmap](/wiki/Beatmap) de otro usuario. Un usuario que está modeando es considerado como un *modder*, y sus posts colectivos en una discusión de un beatmap se denomina como un *mod*.
 
-Si estás aprendiendo a mapear o ya sabes cómo hacerlo, modear es una buena manera para mejorar tus habilidades del mapeo.
+Los Mods son principalmente usados para mejorar beatmaps que apuntan a un [estado Ranked](/wiki/Beatmap/Category#ranked). Debido a que es un proceso detallado y analitico, modear beatmaps tambien es una manera popular de mejorar el entendimiento del beatmapping.
 
-## Método básico de modding
+## Encontrando beatmaps para modear
 
-Primero, descarga un map (Beatmap) desde [Pending](/wiki/Beatmap) o [Works In Progress/Help](/wiki/Beatmap/Category#work-in-progress-and-pending) (WIP). Puede ser una petición (Request) de algún mapper o un map totalmente al azar. Solo necesita no estar rankeado. Por ahora, toma un map totalmente al azar.
+Mientras hay incontables maneras de descubrir beatmaps para modear, los siguientes son algunos métodos comunes:
 
-Luego ve al editor y empieza tu respuesta (mod) en el hilo del Beatmap.
+- La [cola del foro de modding](https://osu.ppy.sh/community/forums/60) es la plataforma usada más activa para encontrar beatmaps para modear y para solicitar mods. Un usuario puede crear su propio hilo, refiriéndose como una *cola*, para que otros usuarios publiquen beatmaps que quieran que sean modeados.
```

Comment:
Could you reformulate this sentance to keep `[Modding queue](https://osu.ppy.sh/community/forums/60)` in english

Reply:
Done